taggle-extension/src/components/Buttons.vue

```diff
@@ -1,25 +1,45 @@
 <template>
   <div class="icon-btn">
-    <v-btn small icon @click="deleteBookmark"><v-icon>delete</v-icon></v-btn>
+    <v-btn v-if="isNotDeletedBookmark" small icon @click="deleteBookmark"><v-icon>delete</v-icon></v-btn>
+    <v-btn v-else small icon @click="addBookmark"><v-icon>get_app</v-icon></v-btn>
```

Comment:
궁금한게
```suggestion
    <v-btn v-if="isNotDeletedBookmark" small icon @click="deleteBookmark">
            <v-icon>delete</v-icon>
    </v-btn>
    <v-btn v-else small icon @click="addBookmark">
            <v-icon>get_app</v-icon>
    </v-btn>
```

이런식으로 엔터로 구분을 안한이유가 길이가 짧기 때문인가요?? 아니면 다른이유가 있나요??
코드를 보다보면 띄우기도 하고 안띄우기도 하는 경우를 봤어서요!
동글만의 기준이 있는건지 궁금하네요!

Reply:
Prettier 교정에만 의존하고 개인적인 의도로 개행을 하거나 그러진 않았어요.